README.md

```diff
@@ -1,2 +1,87 @@
+
+For the purposes of budgetary accountability, the _Dev Team_ will select a Chair and Treasurer. (How the budget is managed is between the _Dev Team_ and the Trustees.) In practice, unlike the _Core Team_ budget which is fairly detailed (e.g. £x on Kickstart, £y on Competition, £z on other) and is approved in advance, _Dev Team_ purchasing will be defined at a higher level (maybe some “infrastructure” budget for a dev server and some dev tools but not £a for Project A, £b for Project B. Unlike the _Core Team_, this will not be a rolling budget, but a “pot” that gets topped up in each accounting period to allow for projects to be budgeted over multiple years.
+
+## Accountability
+
+1. Much like _Core Team_, minutes and reports from the _Dev Team_, and its various sub-groups will be available to view by the general public. Sensitive data, such as budgeting, may be exempt from this.
+2. The _Dev Team_ will submit regular progress reports to the Trustees.
+3. As well as regular volunteers, the _Dev Team_ may contain trustees and members of the _Core Team_, however volunteers should bear in mind any potential conflicts of interest that may arise as a result of this.
+4. Members of the _Dev Team_ should abstain from financial decisions where they are directly involved with the _project group_ of concern. In an event when all members of the _Dev Team_ have to abstain, the decision should be passed to the _Core Team_ or Trustees.
+5. The _Dev Team_ is responsible for the oversight of the various project groups within Student Robotics. The _project groups_ should report their progress regularly to the Dev Team.
+
+## Relationship to the Core Team
+
+- The _Dev Team_ is independent from the _Core Team_, but will work closely with them in certain situations.
+- The _Core Team_ will also communicate priorities for projects to be completed by the next competition cycle. In the event of a Kit Deployment, the _Core Team_ is responsible for budgeting, manufacture, deployment and testing.
+
+## Getting Things Done
+
+For new projects, there is a process to follow to ensure they receive sufficient review and testing before they are implemented. This is shown in the examples below.
+
+For each _project group_ it is recommended that a “Project” is opened and actively maintained with high-level progress updates in the `srobo` GitHub - even the “nice to haves” - if there are good ideas being worked on which get parked here, at least there will be some documentation to pick up on.
+
+Examples of what may be in scope for SR2020: Python 3, new brain board, new IDE.
+
+### Project of significant impact: New brain board
+
+- A (group of) volunteer(s) decide(s) they’d like to work on finding a new single board computer which can serve as the brain board, and port/upgrade software to work on it.
+- New / Updated ticket in srobo/dev-tasks.
```

Comment:
I suggest this repo be the same one used for the dev team minutes, in which case `srobo/dev-team` might be a better name?

Reply:
I note that Core Team minutes go into a core team minutes repo, whereas tasks go into a tasks repo if I am not mistaken. Why should this be different for the dev team?

Reply:
That's not quite correct. The tasks repo contains tasks for the entirety of SR, rather than just to do with competition operations and core team specific tasks.

Tasks specific to the core team _do_ go in the core team minutes repo.

Reply:
There should definitely be a separation from the Core Team's responsibilities. It also prevents spamming other teams with updates that they don't care about.

Reply:
If this tasks repo is separate from https://github.com/srobo/tasks/, how do we co-ordinate development tasks which the Core Team need doing for the current ongoing competition? Would they go into both? (At which point we've got multiple copies of the same task) Or just into one? (At which point we've got multiple places which people need to look at in order to understand what needs doing)